projects/plugins/jetpack/extensions/blocks/blogroll/use-get-site-details.js

```diff
@@ -12,12 +12,21 @@ export default function useGetSiteDetails( { siteURL, subscriptions, enabled = f
 
 	if ( wpcomSite ) {
 		results.unshift( {
-			id: wpcomSite?.ID,
+			blog_id: wpcomSite?.ID,
 			description: wpcomSite?.description,
 			URL: wpcomSite?.URL,
 			site_icon: getSiteIcon( wpcomSite?.logo?.url ),
 			name: wpcomSite?.name,
```

Comment:
Not a blocker, but can we add `is_non_wpcom_site: false` to this for object structure consistency?